api/dps/server.go

```diff
@@ -188,93 +188,44 @@ func (s *Server) GetHeight(_ context.Context, req *GetHeightRequest) (*GetHeight
 // GetTransaction implements the `GetTransaction` function of the generated GRPC
 // server.
 func (s *Server) GetTransaction(_ context.Context, req *GetTransactionRequest) (*GetTransactionResponse, error) {
-	transactionID := flow.HashToID(req.TransactionID)
+	txID := flow.HashToID(req.TransactionID)
```

Comment:
We never really use `id` in the Flow code. Only when we have a field in a struct, it's called `ID`, because it reads `transaction.ID`. Just having `id` is ambiguous at first glance, even if it's not within this context. It could mean a million things, and the type is used for many things, too.